### PR TITLE
New version: simonmichael.hledger version 1.40

### PR DESCRIPTION
**Pull request by Exorcism0666**
Created by [🥟 Dumplings](https://github.com/Exorcism0666/Dumplings) in workflow run [#4131](https://github.com/Exorcism0666/Dumplings/actions/runs/10782198332).

**Logs**
````
Updated: 1.34 → 1.40
Cannot validate argument on parameter 'ChatID'. The argument is null, empty, or consists of only white-space characters. Provide an argument that contains non white-space characters, and then try the command again.
Submitting WinGet manifests
Cannot validate argument on parameter 'ChatID'. The argument is null, empty, or consists of only white-space characters. Provide an argument that contains non white-space characters, and then try the command again.
````
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/172510)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335141
> 2024-09-09 20:16:27.734 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\simonmichael.hledger.1.40\extracted\Pinokio Setup 1.40.exe

Automated error analysis suggests -1978335141 may mean Unable to locate nested installer. Does the `RelativeFilePath` have the previous version number in it?

(Automated response - build 895.)

**Review by stephengillie — DISMISSED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335141
> 2024-09-09 20:16:27.734 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\simonmichael.hledger.1.40\extracted\Pinokio Setup 1.40.exe

Automated error analysis suggests -1978335141 may mean Unable to locate nested installer. Does the `RelativeFilePath` have the previous version number in it?

(Automated response - build 895.)

**Review by stephengillie — COMMENTED** [review]
Please disregard this comment - I clicked too quickly.

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @Exorcism0666,

This PR seems to suggest files from a different package. This is the contents of the `\extracted` folder:
image

File: manifests/s/simonmichael/hledger/1.40/simonmichael.hledger.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PackageVersion: "1.40"
+InstallerType: zip
+FileExtensions:
+- exe
+ReleaseDate: 2024-09-09
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: Pinokio Setup 1.40.exe
```
[the remark this answers is not in the document]
Comment (Clindbergh) [reply manifests/s/simonmichael/hledger/1.40/simonmichael.hledger.installer.yaml:14]: Thank you. I tried to fix it in #186059, but more changes seem to be needed: https://github.com/microsoft/winget-pkgs/pull/186059#issuecomment-2440171164

**Pull request closed without merging**